### PR TITLE
Update About Page

### PR DESCRIPTION
**Pull request by dallasGeorge**
hey, i am sadly unable to test the code due to technical issues with my computer but i hope this sets the ground and its a good merge for the development branch. Not sure about the links to terms of service and privacy policy but for now those were the closest i could find. Feel free to edit it to your likings

### CODE REVIEW

File: core/src/app/halma/AboutScreen.java [hunk 1]
```diff
@@ -16,6 +16,9 @@ public class AboutScreen extends BaseScreen{
     Label linkToMartin;
     Label viewSource;
     Label reportProblem;
+    Label helpWIthTranslation;
```
Comment (CrazyMarvin) [inline core/src/app/halma/AboutScreen.java]: Here is a typo: helpWIthTranslation --> helpWithTranslation

**Comment by CrazyMarvin** [comment]
Thank you so much for this PR!

File: core/src/app/halma/AboutScreen.java [hunk 2]
```diff
@@ -65,6 +73,12 @@ public void clicked(Actor a) {
             link = "https://github.com/mmelnizky";
         if(a == linkToMarvin)
             link = "https://github.com/Crazy-Marvin";
+        if (a == helpWIthTranslation)
+            link = "https://hosted.weblate.org/engage/halma/";
+        if (a == termsOfService)
+            link = "https://github.com/Crazy-Marvin/Halma/blob/development/LICENSE";
+        if (a == privacyPolicy)
+            link = "https://github.com/Crazy-Marvin/Halma/blob/trunk/.github/CONTACT.md";
```
Comment (CrazyMarvin) [inline core/src/app/halma/AboutScreen.java]: This is the link to the Privacy Policy: https://app.getterms.io/view/7FsFk/privacy/en-us

File: core/src/app/halma/AboutScreen.java [hunk 3]
```diff
@@ -65,6 +73,12 @@ public void clicked(Actor a) {
             link = "https://github.com/mmelnizky";
         if(a == linkToMarvin)
             link = "https://github.com/Crazy-Marvin";
+        if (a == helpWIthTranslation)
+            link = "https://hosted.weblate.org/engage/halma/";
+        if (a == termsOfService)
```
Comment (CrazyMarvin) [inline core/src/app/halma/AboutScreen.java]: I thought about Terms of Service and came to the conclusion that they are not needed. Please remove them.
Sorry for the confusion.

**Review by CrazyMarvin — CHANGES_REQUESTED** [review]
Please check my comments

**Comment by dallasGeorge** [comment]
Hey I just fixed the code so it matches with all your comments. I hope everything is ok now and happy new year!

**Pull request merged**